rllib/evaluation/worker_set.py

```diff
@@ -145,8 +145,9 @@ def reset(self, new_remote_workers: List["ActorHandle"]) -> None:
     def stop(self) -> None:
         """Stop all rollout workers."""
         self.local_worker().stop()
+        tids = [w.stop.remote() for w in self.remote_workers()]
+        ray.get(tids)
```

Comment:
Could we wrap this in a try-catch: logger.exception("Failed to stop workers")? Otherwise, an error here could cause the terminate call below to not happen.

Reply:
@ericl Oh good call. I have this now. If you are happy with that I will push.
```python
    def stop(self) -> None:
        """Stop all rollout workers."""
        try:
            self.local_worker().stop()
            tids = [w.stop.remote() for w in self.remote_workers()]
            ray.get(tids)
        except:
            logger.exception("Failed to stop workers")
        finally:
            for w in self.remote_workers():
                w.__ray_terminate__.remote()
```

Reply:
Looks good

Reply:
@ericl I noticed this is still labeled "@author-action-required". Is there something you still need?
I noticed there was a build error but I am not sure it was related to the PR.